packages/rehype-mathjax/chtml.d.ts

```diff
@@ -0,0 +1,21 @@
+import {Plugin} from 'unified'
+
+// http://docs.mathjax.org/en/latest/options/output/chtml.html#the-configuration-block
+interface MathJaxCHtmlOptions {
+  scale?: number
+  minScale?: number
+  matchFontHeight?: boolean
+  mtextInheritFont?: boolean
+  merrorInheritFont?: boolean
+  mathmlSpacing?: boolean
+  skipAttributes?: {[index: string]: boolean}
+  exFactor?: number
+  displayAlign?: 'left' | 'center' | 'right'
+  displayIndent?: string
+  fontURL: string
```

Comment:
is `fontURL` required? In the docs, it seems to indicate there is a default https://docs.mathjax.org/en/latest/options/output/chtml.html#the-configuration-block
Which implies it is optional, is that accurate?

Reply:
The default `fontURL` does not make sense in our case, as we’re processing on the server, so for this project it is required. See also: <https://github.com/remarkjs/remark-math/pull/45/commits/0c4505c44635b6cd3c7fc49b408a985ac68684f4#r422095782>